.changeset/mean-clocks-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/breadcrumb": major
```

Comment:
i think this fix is not major 🤔  at least, minor change...
@illionillion 
do you think this fix deserve major??

Reply:
> Create the changeset with major.
Please derive a new branch from the v2.0 branch and create a pull request for the v2.0 branch, not the main branch, as we plan to release the product in v2.0.0.

sorry, i now found the line at description.
- https://github.com/yamada-ui/yamada-ui/issues/1410